version.php

```diff
@@ -23,7 +23,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2022012001;
+$plugin->version   = 2022120200;
```

Comment:
because we have forked from upstream make this the smaller micro bump you can, and if we can even get by without a version change then even better

Reply:
Good point - I think a micro bump may be required since for e.g. `$PAGE->requires->css` is cached to the plugin version, so if the version doesn't change it might still try to load it from the previous URL and not utilise the new options.

Reply:
Actually on more thought I remember testing before/after the update last week and didn't bump the version so it wouldn't trigger a downgrade error and the caching seemed to update fine.

I'll drop the version bump commit